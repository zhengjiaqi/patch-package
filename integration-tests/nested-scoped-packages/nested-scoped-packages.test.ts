import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "nested-scoped-packages",
  shouldProduceSnapshots: true,
})
