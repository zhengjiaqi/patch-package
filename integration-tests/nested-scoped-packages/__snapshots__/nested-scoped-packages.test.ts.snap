// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test nested-scoped-packages: create the patch 1`] = `
"SNAPSHOT: create the patch
patch-package 0.0.0
â€¢ Creating temporary folder
â€¢ Installing @types/angular@1.6.53 with yarn
â€¢ Diffing your files with clean files
âœ” Created file patches/@microsoft+mezzurite-core++@types+angular+1.6.53.patch

ðŸ’¡ @types/angular is on GitHub! To draft an issue based on your patch run

    yarn patch-package @microsoft/mezzurite-core/@types/angular --create-issue

END SNAPSHOT"
`;

exports[`Test nested-scoped-packages: run patch-package 1`] = `
"SNAPSHOT: run patch-package
patch-package 0.0.0
Applying patches...
@microsoft/mezzurite-core/@types/angular@1.6.53 âœ”
END SNAPSHOT"
`;
